.github/workflows/docker-tagged.yml

```diff
@@ -0,0 +1,38 @@
+name: Docker Tagged Build
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_NAMESPACE: "openfaas"
+      DOCKER_IMAGE_NAME: "nats-connector"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - uses: olegtarasov/get-tag@v2
```

Comment:
We need to avoid third-party actions.

This is one I put together earlier, where we need static binaries plus container images, however it should be similar to here:

https://github.com/openfaas/classic-watchdog/blob/master/.github/workflows/publish.yaml

I think we should have GHCR here too.

Reply:
I agree for something simple like this it's probably not worth the extra risk/hassle for third-party actions.

Re GHCR, is the plan to continue pushing to docker hub and quay.io, as well as adding GHCR?